### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by S3nS3IW00** [PyvesB/AdvancedAchievements#351]
Hello PyvesB!

Can I display more reward line in custom achievement?

Thanks!

**Comment by PyvesB** [comment]
Hello @S3nS3IW00 !

I'm not sure what you mean. Are you talking about the rewards list in the GUI, in the chat, or somewhere else?

Cheers,

Pyves

**Comment by S3nS3IW00** [comment]
Example:

Command:
  Execute: <command>
  Display: <desc1>
  Display: <desc2>

**Comment by PyvesB** [comment]
I see. It's not possible to duplicate the parameters like that, but have you tried using `\n` characters to break lines in a single `Display` message?

**Comment by S3nS3IW00** [comment]
![2018-05-06_11 37 25](https://user-images.githubusercontent.com/22691959/39671956-f2f2117a-5121-11e8-9883-c21394557db4.png)

With \n.

**Comment by PyvesB** [comment]
Fair enough, I now seem to recall that Minecraft doesn't support `\n` in object lores, but does in standard chat messages. 

I'm guessing we could easily support this special case by modifying the `getRewardListing` method in the _RewardParser.java_, as this is what is used to build lores. @supertassu do you want to give it a try and extend your custom reward message implementation? 😄

**Comment by supertassu** [comment]
I'll give it a go.
